pkg/controller/podgc/gc_controller.go

```diff
@@ -97,11 +97,13 @@ func NewPodGCInternal(ctx context.Context, kubeClient clientset.Interface, podIn
 }
 
 func (gcc *PodGCController) Run(ctx context.Context) {
+	logger := klog.FromContext(ctx)
+
 	defer utilruntime.HandleCrash()
 
-	klog.Infof("Starting GC controller")
+	logger.Info("Starting GC controller")
```

Comment:
```suggestion
	klog.FromContext(ctx).Info("Starting GC controller")
```
no need for an extra variable